superset-frontend/src/components/Chart/DrillDetail/DrillDetailPane.tsx

```diff
@@ -302,7 +298,7 @@ export default function DrillDetailPane({
           recordCount={resultsPage?.total}
           usePagination
           loading={isLoading}
-          onChange={(pagination: TablePaginationConfig) =>
```

Comment:
wouldn't we want to leave the annotations to make it more clear on what data is being passed from the function?

Reply:
I took it out because TS can infer this type, so you'll still see `TablePaginationConfig` as the type if you inspect in your IDE and you'll still get the same lint errors as if it were explicitly annotated, and I usually wouldn't include an annotation in those cases to keep the code cleaner.  But, no strong feelings – do you think explicit annotation is helpful here?